service/controller/v15/credential/credential.go

```diff
@@ -62,7 +62,14 @@ func readCredential(k8sClient kubernetes.Interface, obj interface{}) (*v1.Secret
 	}
 
 	credentialName := key.CredentialName(customObject)
+	if credentialName == "" {
+		return nil, microerror.Mask(credentialNameEmpty)
+	}
+
 	credentialNamespace := key.CredentialNamespace(customObject)
+	if credentialName == "" {
+		return nil, microerror.Mask(credentialNamespaceEmpty)
+	}
```

Comment:
Do we have to backport something now to `v14` and so on?

Reply:
If we want to keep some consistency in error returned: yes.